js/server/modules/@arangodb/foxx/manager.js

```diff
@@ -838,20 +838,20 @@ function install (serviceInfo, mount, options = {}) {
       `
     });
   }
-  const tempPaths = _prepareService(serviceInfo, options.legacy);
+  const tempPaths = _prepareService(serviceInfo, Boolean(options.legacy));
   const tempService = FoxxService.create({
     mount,
     basePath: tempPaths.tempServicePath,
     noisy: true,
     options: {
-      development: options.development,
+      development: Boolean(options.development),
       configuration: options.configuration,
       dependencies: options.dependencies
     }
   });
   _install(tempService, tempPaths.tempBundlePath, {
-    setup: options.setup !== false,
-    force: options.force === true
+    setup: options.setup === undefined || Boolean(options.setup),
```

Comment:
This avoids possible bugs if the user provides `0` or `1` instead of booleans, which the HTTP API technically allows although it is not explicitly documented AFAIK. `Boolean` coerces its argument to a boolean value.